app/core.cpp

```diff
@@ -379,6 +382,14 @@ void Core::DialogExportShow()
   }
 }
 
+#ifdef USE_OTIO
+void Core::DialogImportOTIOShow(QList<Sequence*> sequences) {
```

Comment:
Might be nice to change this to `const QList<Sequence*> &sequences` to try saving a deep copy (though Qt might deep copy here anyway when going between threads)